### ISSUE TITLE
Regional Settings Impact Soap Client generation

### ISSUE THREAD

**Issue by SurushS** [dotnet/wcf#2715]
I have an issue on the Wcf Client generator on Dotnet Core projects. When generating the code, if you have your computer regional settings set to something non english ( comma used in decimals or dots). the generated code will not compile. 

If i put my regional settings to dutch, this is the output (will not compile):

```csharp
this.initial_Costs_AmountField = ((decimal)(0,0m));
```

and if i put my regional settings to english (united states) this is the output (does compile):
```csharp
this.initial_Costs_AmountField = ((decimal)(0.0m));
```

Is this a configuration thing or a bug?

**Comment by SurushS** [comment]
Hi @mlacouture . I will see if i can facilitate it to you this week. I have to go back in time a bit and find out which service it was. I'll keep you posted :)

**Comment by mlacouture** [comment]
Hi @SurushS, don't worry, I was able to reproduce the problem. thank you for your help!

**Comment by SurushS** [comment]
> Hi @SurushS, don't worry, I was able to reproduce the problem. thank you for your help!

Great 👍  If there is anything else you need just let me know!

**Comment by dasetser** [comment]
This is now fixed in dotnet-svcutil v1.0.4.
